kind-diag/action.yaml

```diff
@@ -60,5 +66,5 @@ runs:
     - name: Upload artifacts
       uses: actions/upload-artifact@v2
       with:
-        name: logs
+        name: ${{ input.artifact-name }}
```

Comment:
Should be `inputs` rather than `input` looking at https://github.com/chainguard-dev/apko/runs/5807707928?check_suite_focus=true